lib/customEmotionOptions.js

```diff
@@ -0,0 +1,69 @@
+export const customEmotionTypes = [
```

Comment:
these customEmotionTypes need to have the same structure as the initial ones. Please add the field "emotionType" here.

Reply:
refactored